src/document_to_podcast/preprocessing/__init__.py

```diff
@@ -1,20 +1,20 @@
-from .data_loaders import load_pdf, load_txt, load_docx, load_url
-from .data_cleaners import clean_with_regex, clean_html, clean_markdown
+from .data_loaders import load_pdf, load_txt, load_docx, load_url, load_file
+from .data_cleaners import clean_with_regex, clean_html, clean_markdown, markdown_to_text
 
 
 DATA_LOADERS = {
-    ".docx": load_docx,
+    ".docx": load_file,
     ".html": load_txt,
     ".md": load_txt,
-    ".pdf": load_pdf,
+    ".pdf": load_file,
     ".txt": load_txt,
     "url": load_url,
 }
 
 DATA_CLEANERS = {
-    ".docx": clean_with_regex,
+    ".docx": markdown_to_text,
     ".html": clean_html,
     ".md": clean_markdown,
-    ".pdf": clean_with_regex,
+    ".pdf": markdown_to_text,
     ".txt": clean_with_regex,
 }
```

Comment:
Could we also just add more extensions here with the same `load_file` function? For example,
```
    ".json": load_file,
    ".pptx": load_file,
    ".csv": load_file,
```
and


```
    ".json": markdown_to_text,
    ".pptx": markdown_to_text,
    ".csv": markdown_to_text,
```
or is there a limitation I am missing?

Reply:
I've tried it `pptx`, and it works just fine. Not sure about `csv` and `json` though, will add relevant tests and let you know.

Reply:
if it starts becoming too cumbersome to test all of these file formats feel free to skip it, its not a blocker for this PR to be merged :)

Reply:
Thanks! Because it's going to be difficult testing all of these file formats.